### PR TITLE
Document app commands

### PR DESCRIPTION
**Pull request by dpgraham**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: commands-yml/commands/device/app/background-app.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+name: Background App
+short_description: Send the currently running app for this session to the background
+example_usage:
+  java:
+    |
+      driver.runAppInBackground(10);
```
Comment (vmaxim) [inline commands-yml/commands/device/app/background-app.yml]: It should be `driver.runAppInBackground(Duration.ofSeconds(10));`

File: commands-yml/commands/device/app/get-app-strings.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+name: Get App Strings
+short_description: Get app strings
+example_usage:
+  java:
+    |
+      Map<String> appStrings = driver.getAppStringMap("en", "/path/to/file");
```
Comment (vmaxim) [inline commands-yml/commands/device/app/get-app-strings.yml]: `getAppStringMap()` returns `Map<String, String>`

File: commands-yml/commands/device/app/background-app.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+---
+name: Background App
+short_description: Send the currently running app for this session to the background
```
Comment (imurchie) [inline commands-yml/commands/device/app/background-app.yml]: We need a long description to discuss the differences in xctest-based iOS automation.

See:
* https://github.com/appium/appium/issues/7741 
* https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/general.js#L27-L70

File: commands-yml/commands/device/app/close-app.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/appium/app/close
+  method: 'POST'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+  json_parameters:
```
Comment (imurchie) [inline commands-yml/commands/device/app/close-app.yml]: Where do these come from? https://github.com/appium/appium-base-driver/blob/master/lib/mjsonwp/routes.js#L433-L435

File: commands-yml/commands/device/app/get-app-strings.yml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/appium/app/strings
+  method: 'POST'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+  json_parameters:
+    - name: language
```
Comment (imurchie) [inline commands-yml/commands/device/app/get-app-strings.yml]: Do we have no way of indicating required vs. optional payload parameters?

File: commands-yml/commands/device/app/launch-app.yml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/appium/app/launch
+  method: 'POST'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+  json_parameters:
```
Comment (imurchie) [inline commands-yml/commands/device/app/launch-app.yml]: Where are these coming from? https://github.com/appium/appium-base-driver/blob/master/lib/mjsonwp/routes.js#L430-L432

**Review by imurchie — CHANGES_REQUESTED** [review]

File: commands-yml/commands/device/app/get-app-strings.yml (hunk 5 above)
Comment (dpgraham) [reply commands-yml/commands/device/app/get-app-strings.yml]: Not yet. I can add that in a future PR

Comment (dpgraham) [reply commands-yml/commands/device/app/get-app-strings.yml]: I just added 'optional' in parantheses

**Comment by dpgraham** [comment]
Fixes pushed except for the long description.

**Pull request merged**